test/elixir/test/coffee_test.exs

```diff
@@ -0,0 +1,82 @@
+defmodule CoffeeTest do
+  use CouchTestCase
+
+  @moduletag :coffee
+  @headers ["X-Couch-Full-Commit": "false"]
```

Comment:
Not sure if we need this header any more. I ran this test with that header removed, and it also passed.

Reply:
With the new approach, the need for this header became absolute zero. I removed it indeed. Thanks!